src/EnergyPlus/CurveManager.cc

```diff
@@ -2549,11 +2549,11 @@ namespace Curve {
                 thisCurve->GridValueIndex = state.dataCurveManager->btwxtManager.addOutputValues(gridIndex, lookupValues);
 
                 if (normalizeMethod == NM_AUTO_WITH_DIVISOR) {
-                    auto const normalizeTarget = varListNormalizeTargets.at(indVarListName);
+                    auto const &normalizeTarget = varListNormalizeTargets.at(indVarListName);
 
                     bool pointsSpecified = false;
                     bool pointsUnspecified = false;
-                    for (auto value : normalizeTarget) {
+                    for (double value : normalizeTarget) {
```

Comment:
This one will have to sink in, normalizeTarget is an object reference so value needs? to be a copy

Reply:
`normalizeTarget` is an object reference, but it references an array of `double`s, so individual value in that range loop will be of type `double`. And scalars should not be used `by reference`.